src/cfnlint/__init__.py

```diff
@@ -655,28 +655,33 @@ def get_values(self, obj, key, path=[]):
             return None
         if isinstance(value, (dict)):
             if len(value) == 1:
+                is_condition = False
                 for obj_key, obj_value in value.items():
                     if obj_key in cfnlint.helpers.CONDITION_FUNCTIONS:
                         results = self.get_condition_values(obj_value, path[:] + [obj_key])
                         if isinstance(results, list):
                             matches.extend(results)
-                    else:
-                        result = {}
-                        result['Path'] = path[:] + [obj_key]
-                        result['Value'] = obj_value
-                        matches.append(result)
+
+                if not is_condition:
```

Comment:
You set is_condition statically above and I'm not seeing a path where is_condition can go "True" in this function .  Is this just to match the general pattern in the files below?